dbt/models/iasworld/schema/iasworld.pardat.yml

```diff
@@ -259,6 +259,14 @@ sources:
             description: '{{ doc("shared_column_year") }}'
           - name: tieback
             description: '{{ doc("shared_column_tieback_key_pin") }}'
+            tests:
+              - expression_is_true: 
+                  name: iasworld_pardat_tieback_does_not_contain_hyphen
+                  expression: |
+                    not like '%-%'
+                  additional_select_columns: 
+                    - parid
+                    - taxyr
```

Comment:
**[Nitpick]** Let's add `meta` attributes to make sure this test gets displayed correctly in the output workbook. We'll need to set these two attributes under the `meta` key:

* `category: incorrect_values` (necessary because `expression_is_true` is very general and so cannot be automatically assigned to a specific test category by [the code in `transform_dbt_test_results.py`](https://github.com/ccao-data/data-architecture/blob/14095e2a9bb15106ae95807f2e8302a34a3b7e45/.github/scripts/transform_dbt_test_results.py#L1113-L1131))
* `description: tieback should not contain hyphens`

For reference, here's an example of a test that defines these two attributes, albeit with different values:

https://github.com/ccao-data/data-architecture/blob/14095e2a9bb15106ae95807f2e8302a34a3b7e45/dbt/models/iasworld/schema/iasworld.pardat.yml#L77-L96